packages/wallets/src/components/Base/ModalStepWrapper/ModalStepWrapper.scss

```diff
@@ -3,13 +3,17 @@
     border-radius: 0.8rem;
 
     @include mobile {
-        height: calc(100vh - 4rem);
+        height: calc(100svh - 4rem);
```

Comment:
I couldn't use `dvh`, it was restricted by the linter, `svh` should work just fine though 👀 :
<img width="559" alt="Screenshot 2023-11-20 at 1 42 41 PM" src="https://github.com/binary-com/deriv-app/assets/103016120/e19aad95-5457-4973-b1ce-c862352c90aa">